roles/download/defaults/main.yml

```diff
@@ -34,84 +34,57 @@ download_delegate: "{% if download_localhost %}localhost{% else %}{{groups['kube
 # Arch of Docker images and needed packages
 image_arch: "{{host_architecture | default('amd64')}}"
 
-# Versions
-kube_version: v1.12.3
-kubeadm_version: "{{ kube_version }}"
-etcd_version: v3.2.24
-
 # kubernetes image repo define
 kube_image_repo: "gcr.io/google-containers"
 
-# TODO(mattymo): Move calico versions to roles/network_plugins/calico/defaults
-# after migration to container download
-calico_version: "v3.1.3"
-calico_ctl_version: "v3.1.3"
+# Versions
+addon_resizer_version: "1.8.3"
+busybox_version: "1.29.2"
 calico_cni_version: "v3.1.3"
+calico_ctl_version: "v3.1.3"
 calico_policy_version: "v3.1.3"
 calico_rr_version: "v0.6.1"
-
-flannel_version: "v0.10.0"
-flannel_cni_version: "v0.3.0"
-
+calico_version: "v3.1.3"
+cephfs_provisioner_version: "v2.1.0-k8s1.11"
+cert_manager_version: "v0.5.2"
+cilium_version: "v1.3.0"
 cni_version: "v0.6.0"
-
-weave_version: 2.5.0
-pod_infra_version: 3.1
+contiv_etcd_init_version: "latest"
+contiv_init_version: "latest"
+contiv_ovs_version: "latest"
 contiv_version: 1.2.1
-cilium_version: "v1.3.0"
+coredns_version: "1.2.6"
+dashboard_version: "v1.10.0"
+dnsautoscaler_version: 1.3.0
+dnsmasq_version: "2.78"
+dnsmasqautoscaler_version: 1.1.2
+etcd_version: v3.2.24
+flannel_cni_version: "v0.3.0"
+flannel_version: "v0.10.0"
+helm_version: "v2.11.0"
+ingress_nginx_controller_version: "0.21.0"
 kube_router_version: "v0.2.1"
+kube_version: v1.12.3
+kubeadm_version: "{{ kube_version }}"
+kubedns_version: 1.14.13
+local_volume_provisioner_version: "v2.1.0"
+metrics_server_version: "v0.3.1"
 multus_version: "v3.1.autoconf"
+netcheck_version: "v1.0"
+nginx_version: "1.13"
+pod_infra_version: "3.1"
+registry_proxy_version: "0.4"
+registry_version: "2.6"
+socat_image_version: "latest"
+weave_version: 2.5.0
 
 # Download URLs
 kubeadm_download_url: "https://storage.googleapis.com/kubernetes-release/release/{{ kubeadm_version }}/bin/linux/{{ image_arch }}/kubeadm"
-hyperkube_download_url: "https://storage.googleapis.com/kubernetes-release/release/{{ kube_version }}/bin/linux/amd64/hyperkube"
+kubelet_download_url: "https://storage.googleapis.com/kubernetes-release/release/{{ kubeadm_version }}/bin/linux/{{ image_arch }}/kubelet"
+kubectl_download_url: "https://storage.googleapis.com/kubernetes-release/release/{{ kubeadm_version }}/bin/linux/{{ image_arch }}/kubectl"
 etcd_download_url: "https://github.com/coreos/etcd/releases/download/{{ etcd_version }}/etcd-{{ etcd_version }}-linux-amd64.tar.gz"
 cni_download_url: "https://github.com/containernetworking/plugins/releases/download/{{ cni_version }}/cni-plugins-{{ image_arch }}-{{ cni_version }}.tgz"
 
-# Checksums
-hyperkube_checksums:
```

Comment:
Why would we remove this? 
That was quite some work to get that, and it add significant security verification of the binaries installed. 
I would always choose toward fixing issues to add more security and verification than fixing them by removing.

Reply:
Checksum validation only works for the version defined and amd64